### DISCUSSION TITLE
Should ratings with no date be possible?

### DISCUSSION THREAD

**Discussion by tomtjes** [TandoorRecipes/recipes#1940] [Ideas]
Hi,

I just wanted to put this up for discussion. I'd like to be able to rate recipes without attaching a date to the rating. Currently, all cook log entries (where ratings are saved) must have a date. But as I enter recipes into Tandoor that I've made before, I would like to rate them immediately, without knowing when I last made them.

On the other hand there's probably a good reason why this is not currently possible... Let's hear some pros and cons!

**Comment by vabene1111** [comment]
the whole rating, commenting and logging system is something that needs work, it will likely also include separating ratings from logging.
